foundation/foundation-upgrade/src/main/resources/migrations.json

```diff
@@ -3,6 +3,15 @@
     "group": "v1-8-0-migrations",
     "type": "ordered",
     "migrations": [
+      {
+        "name": "python-linting-migration",
+        "implementation": "com.boozallen.aissemble.upgrade.migration.v1_7_0.PythonLintingMigration",
```

Comment:
C: Update package from `v1_7_0` to `v1_8_0`

Reply:
Updated, great catch Emily! Thank you.